source/community/release_notes/quince.rst

```diff
@@ -27,14 +31,36 @@ Instructor Experiences
 Administrators & Operators
 **************************
 
+* `feat: feature flag to disable Advanced Settings <https://github.com/openedx/edx-platform/pull/32015>`__
+
+Which MFEs will be included?
+
+* `frontend-app-learner-record: <https://github.com/openedx/frontend-app-learner-record>`__
+* `frontend-app-learner-dashboard: <https://github.com/openedx/frontend-app-learner-dashboard>`__
+
+New Waffle Flagts:
+
+* `feat: add Waffle Flag to disable resetting self-paced deadlines by learners: <https://github.com/openedx/edx-platform/pull/32148>`__
+* `feat: adds waffle flag for show notifications tray: <https://github.com/openedx/edx-platform/pull/32451>`__
+* `feat: add course waffle flag for learner assistant: <https://github.com/openedx/edx-platform/pull/32657>`__
```

Comment:
Ghassan made a list of all the waffle flag and feature settings changes between palm and quince in https://github.com/openedx/wg-build-test-release/issues/316 I'm not sure why these didn't come up in your review of commits. I think Ghassan generates his list by crawling the source code for the code annotations we use to identify settings and flags. 

In any case, I prefer to link them to their documentation instead of their PRs. You should be able to find [settings](https://docs.openedx.org/projects/edx-platform/en/latest/references/settings.html) and [feature toggles](https://docs.openedx.org/projects/edx-platform/en/latest/references/featuretoggles.html) in the reference section of docs.edx.org